content/en/seachat/seachat-manual/02-create-agent/01-create-new-agent.md

```diff
@@ -107,5 +107,37 @@ Unlike starting from scratch, a use case comes with predefined instructions prep
 
 <br/>
 
+## Agent Duplication
```

Comment:
could you make this a new section?

![image](https://github.com/seasalt-ai/wiki/assets/543547/2cda3a2f-dae8-42e8-91a1-c7e5cd45e44e)

Agent Duplication sits right under "Create New Agent" on the left nav bar.